bin/os_main.tf

```diff
@@ -0,0 +1,103 @@
+#Openstack credentials and instance settings, SSH Variables and current image version
+variable username {}
+
```

Comment:
Indeed, might have accidentally pressed enter there! No idea otherwise. Correcting it.